nautobot/extras/tests/test_api.py

```diff
@@ -2179,6 +2181,65 @@ def setUpTestData(cls):
         Tag.objects.create(name="Tag 2", slug="tag-2")
         Tag.objects.create(name="Tag 3", slug="tag-3")
```

Comment:
Don't these need to have a content_types value to be valid now?

Reply:
TODO: Add content types. Reason: The data behind the 1.2 API call will have content types associated, even though the 1.2 API won't show it.